### ISSUE TITLE
Making an website using this metabase

### ISSUE THREAD

**Issue by SaurabhHorizon** [cat-milk/Anime-Girls-Holding-Programming-Books#373]
Can i built an website with this database and organise every every image by the language where peoples can also easily download these images?

**Comment by Caslus** [comment]
it has been done before, i'm pretty sure you can go for it

**Comment by ThamognyaKodi** [comment]
Yes, I think a simple GitHub page would be enough, but the bottom line is a website would really be helpful.
